{"CVE Code": "CVE-2025-48432", "Description": "An issue was discovered in Django 5.2 before 5.2.2, 5.1 before 5.1.10, and 4.2 before 4.2.22. Internal HTTP response logging does not escape request.path, which allows remote attackers to potentially manipulate log output via crafted URLs. This may lead to log injection or forgery when logs are viewed in terminals or processed by external systems.", "NVD Link": "https://nvd.nist.gov/vuln/detail/CVE-2025-48432", "Vulnerability Categories": "N/A", "Affected Products": "N/A", "EPSS Score": "0.05% Probability of exploitation activity in the next 30 days\n~ 17 % Percentile, the proportion of vulnerabilities that are scored at or less", "EPSS History": "\n #  Date  Old EPSS Score  New EPSS Score  Delta (New - Old) \n Could not find any EPSS score change records for this CVE ", "CVSS Scores": "4.0", "CWE": [{"CWE_ID": "CWE-117 Improper Output Neutralization for Logs", "CWE_link": "/cwe-details/117/Improper-Output-Neutralization-for-Logs.html"}], "Reference": [{"ref_link": "https://groups.google.com/g/django-announce", "ref_desc": "django-announce - Google GroupsCVEs referencing this url", "ref_cve_link": "https://www.cvedetails.com/reference-url-info/7Kv1mhz3CyWuc-iy_G2zgXiEkSU.html"}, {"ref_link": "https://www.djangoproject.com/weblog/2025/jun/04/security-releases/", "ref_desc": "Django security releases issued: 5.2.2, 5.1.10, and 4.2.22 | Weblog | Django", "ref_cve_link": "N/A"}, {"ref_link": "https://docs.djangoproject.com/en/dev/releases/security/", "ref_desc": "Archive of security issues | Django documentation | DjangoCVEs referencing this url", "ref_cve_link": "https://www.cvedetails.com/reference-url-info/InwfdzuHdb1agC8uMwH31EJtlGA.html"}, {"ref_link": "http://www.openwall.com/lists/oss-security/2025/06/04/5", "ref_desc": "oss-security - CVE-2025-48432: Django: Potential log injection via unescaped request path", "ref_cve_link": "N/A"}]}